src/sentry/static/sentry/app/views/organizationIntegrations/sentryAppInstallations.jsx

```diff
@@ -45,6 +48,17 @@ class SentryAppInstallations extends React.Component {
       const redirectUrl = addQueryParamsToExistingUrl(app.redirectUrl, queryParams);
       window.location.assign(redirectUrl);
     }
+
+    //hack for split
+    if (organization.slug === 'split') {
```

Comment:
Is this a short term thing?

Reply:
@markstory Yes. We are working on a more permanent solution but this will unblock things for Split so they can build the integration.

Reply:
Created https://getsentry.atlassian.net/browse/API-448